### ISSUE TITLE
House building

### ISSUE THREAD

**Issue by Floppy** [openpolitics/manifesto#429]

**Comment by Xyleneb** [comment]
One idea on this: seize any lot that has been vacant for four decades. If the private sector doesn't want any utility, then the public will take it.

**Comment by Floppy** [comment]
Hm, nice idea. Pop it in an edit?

**Comment by philipjohn** [comment]
Yep, sounds interesting. We currently have [this on empty properties](https://openpolitics.org.uk/manifesto/housing.html#empty-properties):

> Empty Properties
> Residential properties that have been empty for more than 12 months will automatically attract a 100% increase in council tax.
> 
> A landlord of a property that has been unoccupied for 12 months or more can, once the property has been occupied for 12 months or more, claim a 6 month rebate on the council tax paid before the property was occupied.
> 
> Government-backed lending schemes will be offered to landlords with unoccupied residential properties that require refurbishment or repair in order to get them back into use.
